subxt/src/extrinsic/mod.rs

```diff
@@ -15,6 +15,32 @@
 // along with subxt.  If not, see <http://www.gnu.org/licenses/>.
 
 //! Create signed or unsigned extrinsics.
+//!
+//! This modules exposes the extrinsic's parameters and the ability to sign an extrinsic.
+//!
+//!
+//! An extrinsic is submitted with an "signed extra" and "additional" parameters, which can be
+//! different for each chain. The trait [ExtrinsicParams] is at the core of an extrinsic parameter.
+//!
+//!
+//! The structure [BaseExtrinsicParams] is a base implementation of the trait which
+//! configures most of the "signed extra" and "additional" parameters as needed for
+//! Polkadot and Substrate nodes. Only the shape of the tip payments differs, leading to
+//! [SubstrateExtrinsicParams] and [PolkadotExtrinsicParams] structs which pick an
+//! appropriate shape for Substrate/Polkadot chains respectively.
+//!
+//! For this implementation:
+//! - "signed extra" contains:
+//!     - sp_runtime::Era: This is utilized to determine the longevity of a transaction.
+//!     - nonce: Account index (aka nonce) that stores the number of previous transactions
+//!         associated with a sender account. This is utilized to avoid replay attacks.
+//!     - tip: A tip payment for including the transaction in the block.
+//! - "additional" contains:
+//!     - spec_version: The version of the runtime specification of the node.
+//!     - transaction_version: The version of the extrinsic interface. This allows hardware
+//!         wallets to know which transactions can be safely signed.
+//!     - genesis_hash: The hash of the Genesis block.
+//!     - mortality_checkpoint: The block hash after which the transaction becomes valid.
```

Comment:
I'm happy to keep this, but aware that it can easily fall out of sync with reality if these things change :)

Reply:
In that case, we could remove this for now. It's hard to notice when those things change, even so, remember to update the comment. We could add it back later when things are stable enough.

Reply:
Sounds good to me; I think what you wrote above was sufficient anyway :)